### DISCUSSION TITLE
pgqr extension

### DISCUSSION THREAD

**Discussion by Antiokh** [supabase/supabase#19234] [Feature Requests]
Can you please add a pgqr extension from https://github.com/AbdulYadi/pgqr?

It generates qr codes. It would be better to generate SVG qr codes, but I haven't found anything like this.

**Comment by riderx** [comment]
he can add it to dbdev: https://supabase.github.io/dbdev/

**Reply by Antiokh** [reply]
It's last updated 5 years ago. Maybe you know someone who is familiar with dbdev?
